compiler/src/iree/compiler/DispatchCreation/FusionUtils.cpp

```diff
@@ -97,4 +101,35 @@ bool areFusableAsElementwiseOps(MLIRContext *context, OpOperand *fusedOperand,
   return true;
 }
 
+// Returns true when an operation `op` is horizontal to `currGroup` when
+// considering the program slice between `seedOp` and `op`.
```

Comment:
nit: move the comment to header file (i.e., `FusionUtils.h`).

Reply:
Looks like a duplicate of what's in the header, so deleted it